code/development-env/datalabs-internal-setup.md

```diff
@@ -57,20 +65,22 @@ If you are able to use one of these and still set the `--default-ssl-certificate
 ## Install ingress manifests
 
 Make the following changes to [`./config/manifests/minikube-proxy.yml`](./config/manifests/minikube-proxy.yml):
+
 - Update all values referring to `datalabs.localhost` to `datalabs.internal`
 - Ensure the `externalName` field of both the `datalab-app-proxy` and `datalab-api-proxy` services is the IP at which a service running within minikube can access services running on your machine's `localhost`.
   This is expected to be `10.0.2.2` when running minikube with Virtualbox.
   This assumes that both the app and api are running and can be accessed on `localhost`.
 
 Once the changes have been made, apply the manifests using
+
 ```bash
-kubectl apply -f ./config/manifests/minikube-proxy.yaml
+kubectl apply -f ./config/manifests/minikube-proxy.yml
 ```
 
 ## Start Minikube tunnel
 
 At this point, the minikube cluster should be configured correctly in order to receive requests.
-However, the entry point for requests into the cluster (the `LoadBalancer` service `ingress-nginx-controller`) doesn't have an externally accessible IP address as can be seen when running the following command (the `EXTERNAL-IP` column should read `<none>`).
+However, the entry point for requests into the cluster (the `LoadBalancer` service `ingress-nginx-controller`) doesn't have an externally accessible IP address as can be seen when running the following command (the `EXTERNAL-IP` column should read `<pending>`).
```

Comment:
I'm pretty sure mine was `<none>` here so might be worth noting that it could be either `<none>` or `<pending>`

Reply:
Okay